test/libsolidity/semanticTests/enums/enum_explicit_overflow.sol

```diff
@@ -26,8 +26,8 @@ contract test {
 // compileViaYul: also
 // ----
 // getChoiceExp(uint256): 2 -> 2
-// getChoiceExp(uint256): 3 -> FAILURE, hex"4e487b71", 33 # These should throw #
-// getChoiceFromSigned(int256): -1 -> FAILURE, hex"4e487b71", 33
-// getChoiceFromMax() -> FAILURE, hex"4e487b71", 33
+// getChoiceExp(uint256): 3 -> FAILURE, hex"4e487b71", 0x21 # These should throw #
+// getChoiceFromSigned(int256): -1 -> FAILURE, hex"4e487b71", 0x21
+// getChoiceFromMax() -> FAILURE, hex"4e487b71", 0x21
```

Comment:
Should I add this change to #11277? I think it's neither bad nor good so we could keep this one.